### PR TITLE
Added documentation for proposed new sensor component weightedaverage.

### PR DESCRIPTION
**Pull request by stafbulp**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24247

## Checklist:

- [ x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.weightedaverage.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: "Weighted Average Sensor"
+description: "Instructions on how to add weighted average sensors within Home Assistant."
+date: 2019-06-01 01:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```
Comment (klaasnicolaas) [inline source/_components/sensor.weightedaverage.markdown]: doubt if you can use the HA logo for this

File: source/_components/sensor.weightedaverage.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ha_category:
+  - Sensor
+ha_release: 0.93
```
Comment (klaasnicolaas) [inline source/_components/sensor.weightedaverage.markdown]:
```suggestion
ha_release: 0.95
```

File: source/_components/sensor.weightedaverage.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ha_iot_class: Local Push
+ha_qa_scale: internal
+---
+
+The `weightedaverage` sensor calculates an average value from another sensor, that is weighted according to the duration for each individual sensor value.
+This is useful when the source sensor state is updated at uneven intervals, typically a temperature sensor that is only updated when the temperature changes. The weighted average takes into account the 'weight' (i.e. the duration) for each temperature.
+The sensor also records the maximum and minimum sensor values for the given duration, as well as the top rates at which the sensor value has ascended or descended.
+
+
+## {% linkable_title Configuration %}
```
Comment (klaasnicolaas) [inline source/_components/sensor.weightedaverage.markdown:24]: Please add also the [configuration](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration)

File: source/_components/sensor.weightedaverage.markdown [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+To add the weighted average sensor to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
```
Comment (klaasnicolaas) [inline source/_components/sensor.weightedaverage.markdown:89]: In a first example only the required variables to set up this integration

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/sensor.weightedaverage.markdown (hunk 1 above)
Comment (stafbulp) [reply source/_components/sensor.weightedaverage.markdown]: OK, what should I use instead? Can I just skip the logo?

Comment (klaasnicolaas) [reply source/_components/sensor.weightedaverage.markdown]: you can skip it but it doesn't look nice in the components overview. Pherhaps just a picture of a scale/calculator?

**Comment by klaasnicolaas** [comment]
Please rename the file to: `weightedaverage.markdown`

File: source/_components/sensor.weightedaverage.markdown [hunk 5]
```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: "Weighted Average Sensor"
+description: "Instructions on how to add weighted average sensors within Home Assistant."
+date: 2019-06-01 01:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
```
Comment (klaasnicolaas) [inline source/_components/sensor.weightedaverage.markdown:10]:
```suggestion
ha_category:
  - Sensor
  - Utility
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Pull request closed without merging**